src/components/Conviction.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Button, Checkbox, Icon, Input, Select, Table } from 'semantic-ui-react';
+
+import CrimeClassification from '../logic/type/CrimesClassifications';
+
+const convictionClassificationOptions = [
+  {
+    key: CrimeClassification.FELONY_CLASS_B,
```

Comment:
The legal aid lawyer mentioned that there are certain convictions that are "unclassified" from a while back. We'll need an option for an "unknown" option that pushes a null value to the calculator.